src/content/contributing/style-guide/index.md

```diff
@@ -53,40 +82,49 @@ People, businesses, and other organizations who want to understand Ethereum's va
 - "I work at a business that is beginning an Ethereum related project, and want to learn more"
 - "I want to understand the differences between private Ethereum chains, consortium chains, and the public Ethereum Mainnet"
 - "I want to know the current status of Ethereum - how long has it been in production, how much usage it has, what's the direction of new development - to decide if I am confident to build my project on top of it"
+</p>
+</details>
 
-## Objectivity {#objectivity}
-
-Ethereum.org documentation (and content at large) aims to maintain a credibly neutral source of truth to inform readers about Ethereum and its ecosystem. Some examples of things that we don't want in the content on ethereum.org:
+## Best practices {#best-practices}
 
-**Grand, unverifiable claims about Ethereum or adjacent technologies**
+**Style**
 
-> e.g. _"Ethereum will take over the world because..."_
+- [ ]  Focus on the advantages for the user instead of explaining technical details about the system
```

Comment:
I think these will get rendered as tickable-boxes? Does it make sense to have those here?

Reply:
Would have to double check this actually... not sure we have support for markdown checkboxes, since we don't actually use this syntax anywhere else in the repo right now